src/lib/serviceRoutinesV2/patchEntity.cpp

```diff
@@ -71,6 +71,12 @@ std::string patchEntity
   eP->id = compV[2];
   eP->type = ciP->uriParam["type"];
 
+  if (forbiddenIdChars(ciP->apiVersion, eP->id.c_str() , NULL))
+  {
+    OrionError oe(SccBadRequest, "invalid character in URI");
+    return oe.render(ciP, "");
+  }
+
```

Comment:
This seems like something that should be done for **all** v2 requests (perhaps v1 also),
not only PATCH /v2/entities/<eid> ...
Should the check be on a higher level?
Somewhere in rest.cpp ...